adapters/Unity/UnityAdapter/GADMAdapterUnity.m

```diff
@@ -60,7 +63,22 @@ - (void)stopBeingDelegate {
   if (_bannerAd != nil) {
     [_bannerAd stopBeingDelegate];
   }
-  [[GADMAdapterUnitySingleton sharedInstance] stopTrackingDelegate:self];
+}
+
+- (void)initializeWithGameID:(NSString *)gameID {
+    if ([UnityAds isInitialized]) {
+        return;
+    }
+    // Metadata needed by Unity Ads SDK before initialization.
+    UADSMediationMetaData *mediationMetaData = [[UADSMediationMetaData alloc] init];
+    [mediationMetaData setName:kGADMAdapterUnityMediationNetworkName];
+    [mediationMetaData setVersion:kGADMAdapterUnityVersion];
+    [mediationMetaData set:@"adapter_version" value:[UnityAds getVersion]];
+    [mediationMetaData commit];
+    
+    // Initializing Unity Ads with |gameID|.
+    [UnityAds initialize:gameID testMode:NO];
```

Comment:
it should initialize with enable per placement load.

Reply:
Sure, I'll do that. I see the former implementation doesn't have that parameter included.

Reply:
are you sure? I'm looking here https://github.com/googleads/googleads-mobile-ios-mediation/blob/ad67d3341543e48e5cc5bebda588a95798acbfdf/adapters/Unity/UnityAdapter/GADMAdapterUnitySingleton.m#L67

Reply:
Oh. My bad, I must've missed it! I implemented it now.